assignment0_bibek/assignment0.md

```diff
@@ -0,0 +1,30 @@
+# Assignment 0
```

Comment:
You may want to check how this markdown file is rendered. The hashmarks will make all the lists into headings.